docker/addons/vault/.gitignore

```diff
@@ -0,0 +1 @@
+data
```

Comment:
remove this file from PR

Reply:
Why remove this file? 

The `init.sh` script creates a `data/secrets` file, along with the certificates, so this is a perfectly valid usage o Git ignore.

Not having this will lead to Git always having files in its index that should not be under version control.

Reply:
ok, my bad